test/opentelemetry-metrics/fixtures.test.js

```diff
@@ -153,26 +153,26 @@ async function checkHasPrometheusMetrics(t) {
 
 // ---- tests
 
-// We need to `npm install` for a first test run.
+// We need to `npm ci` for a first test run.
 const haveNodeModules = fs.existsSync(path.join(fixturesDir, 'node_modules'));
 tape.test(
-  `setup: npm install (in ${fixturesDir})`,
+  `setup: npm ci (in ${fixturesDir})`,
```

Comment:
There is a similar test in `activation-method.test.js`. Should we change it too?

https://github.com/elastic/apm-agent-nodejs/blob/bcd0fa684f72edc899cfe474da07e35dc6c92156/test/activation-method/activation-method.test.js#L28

Reply:
That fixtures dir doesn't have a package-lock.json, so cannot use `npm ci`.  And the only dep there is the agent itself:

```
[./test/activation-method/fixtures (git:main)]
% cat .npmrc
package-lock=false

[./test/activation-method/fixtures (git:main)]
% cat package.json
{
  "name": "activation-method-fixtures",
  "version": "1.0.0",
  "private": true,
  "dependencies": {
    "elastic-apm-node": "file:../../.."
  }
}
```

So no need for `npm ci`.